widgets/src/components/common/form/widgets/SelectWidget.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import InputBase from '@material-ui/core/InputBase';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+
+// refer to Material UI on custom select styling https://material-ui.com/components/selects/#customized-selects
```

Comment:
not sure if this is needed

Reply:
it has this tricky logic of using input field instead of select, so left the comment for an easy intro